src/timeline-window.ts

```diff
@@ -41,10 +41,14 @@ const DEFAULT_PAGINATE_LOOP_LIMIT = 5;
 
 interface IOpts {
     windowLimit?: number;
+    // A label used in the logs to namespace and differentiate the timelines.
+    //ex. main timeline from the thread timeline.
+    contextLabel?: string;
```

Comment:
TODO: Remove as this turned out to not be necessary and we can log everything from `TimelinePanel` in https://github.com/matrix-org/matrix-react-sdk/pull/8502